packages/app/src/debug/DebugEmptyView.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="flex flex-col my-45px items-center">
+    <div class="flex flex-col items-center justify-evenly">
+      <div><i-cy-box-open_x48 class="icon-dark-gray-500 icon-light-indigo-100" /></div>
+      <div class="flex flex-col mx-[20%] mt-25px mb-20px items-center">
+        <div class="font-medium my-5px text-center text-gray-900 text-18px">
+          {{ title }}
+        </div>
+        <div class="font-normal my-5px text-center leading-relaxed text-16px text-gray-600">
+          {{ description }}
+        </div>
+      </div>
+      <slot name="cta" />
+    </div>
+    <div class="flex flex-col my-40px w-full items-center">
+      <div class="border-t border-l border-r rounded-t-md bg-gray-50 border-0 border-gray-100 py-20px px-15px w-[75%]">
+        <div class="bg-white border rounded-md flex border-gray-100 w-max p-5px text-14px text-gray-700">
+          <div><i-cy-status-failed_x12 /></div>
+          <div class="mx-1">
+            -
+          </div>
+          <div
+            v-if="exampleTestName"
+            class="mx-1 pb-1px text-gray-100"
+          >
+            |
```

Comment:
This is just intended as a divider, right? Seems like a job for CSS rather than a text "pipe" since it will end up selectable